### PR TITLE
adds autoplay functionality

### PR DESCRIPTION
**Pull request by inderpreetsingh**
Adds auto play functionality
* pauses and plays whenever the autoplay is clicked
* when in autoplay you can click on any panktee and it will start autoplaying from there
* goes hand in hand with https://github.com/KhalisFoundation/sttm-desktop/pull/98
autoplay

### CODE REVIEW

File: js/settings.json [hunk 1]
```diff
@@ -68,7 +68,20 @@
         "options": {
           "akhandpaatt": "Akhand Paatth View (Beta)",
           "larivaar": "Larivaar",
-          "left-align": "Left-Align"
+          "left-align": "Left-Align",
+          "autoplay":"Auto Play"
```
Comment (navdeepsinghkhalsa) [inline js/settings.json]: Please add space

File: js/settings.json [hunk 2]
```diff
@@ -68,7 +68,20 @@
...
+        }
+      },
+      "autoplay-options": {
+        "title": "Autoplay Options",
+        "type": "range",
+        "options": {
+          "autoplayTimer": {
+            "title": "Autoplay Time",
```
Comment (navdeepsinghkhalsa) [inline js/settings.json]: Perhaps "Delay (seconds)" would be intuitive. What do you think?

Comment (inderpreetsingh) [reply js/settings.json]: I think, you are right 👏 . Shall be done!

File: js/settings.json (hunk 1 above)
Comment (inderpreetsingh) [reply js/settings.json]: Aaah! Apologies 😭 Still not in habit. :| Fixing it.

**Comment by inderpreetsingh** [comment]
@navdeepsinghkhalsa aah! I got it. For checking delay, right? I think I can do it from `userPrefs` store rather than from DOM. I will create another issue regarding this enhancement and will get that done.

**Comment by navdeepsinghkhalsa** [comment]
@inderpreetsingh I will pass this now, but please create an issue as an enhancement and perhaps you can do it after the repositories are merged?

**Pull request merged**